code/modules/mob/mob_helpers.dm

```diff
@@ -104,6 +104,10 @@ mob/proc/remove_internal_organ()
 			return 1
 	return 0
 
+/proc/ghostCheck(A) //Ghostbusters. Returns 1 if it's a ghost, returns 0 if they're an adminghost or otherwise
```

Comment:
Is there a reason this is a proc instead of a macro?
Also, the name could be more descriptive so one doesn't need to see the code or comment to glean its function.